kernel/USER_GUIDE.md

```diff
@@ -91,57 +91,85 @@ while(fileIter.hasNext()) {
   FilteredColumnarBatch scanFileColumnarBatch = fileIter.next();
   CloseableIterator<Row> scanFileRows = scanFileColumnarBatch.getRows();
```

Comment:
Remove this line? We define `scanFileRows` twice -- here and in the try statement